src/Factory/RequestFactory.php

```diff
@@ -63,15 +60,13 @@ public function __construct(
         string $contentType,
         string $formatType,
         EncoderInterface $encoder,
-        array $defaultBodyContent = [],
```

Comment:
Removing this is a BC break

Reply:
This class isn't used directly in our docs, so I thought it was OK. See https://github.com/supportpal/api-client-php/wiki/Unsupported-Endpoints

Reply:
Fair point :+1: